src/services/app/user.ts

```diff
@@ -23,6 +23,8 @@ export default class UserService {
     public static appUserThroughTrust = models.appUserThroughTrust;
     public static appMediaContent = models.appMediaContent;
     public static appMediaThumbnail = models.appMediaThumbnail;
+    public static storyContent = models.storyContent;
```

Comment:
TBH, I've been trying to remove this and instead use `models.storyContent` (same for other models) within each method. Any opinion regarding this? My reasoning was that, at some point, these instances were not used anywhere on the code. Any opinion?

Reply:
Ahh, seems to be outdated. Sorry :facepalm: